src/todosRepository.js

```diff
@@ -0,0 +1,16 @@
+import * as ko from 'knockout';
+import { Todo } from './todo.model.js';
+
+export const todosRepository = {
+    load() {
+        console.log(window.localStorage);
+        let list = JSON.parse(localStorage.getItem('todoList'));
+
+        return list === null ? [] : list.map((todo) => new Todo(todo.value, todo.checked)) ;
+    },
+
+    save(uploadedList) {
+        const list = ko.toJS(uploadedList);
+        localStorage.setItem('todoList', JSON.stringify(list));
```

Comment:
Move name of the localStorage property to constant or class property and reuse it in both load/save. Right now it's a magical string that can be different on read/write